packages/odyssey/src/scss/base/_typography-text.scss

```diff
@@ -80,8 +86,39 @@ ins::after {
   content: attr(data-a11y-end);
 }
 
-mark { background: cv('warning', 'lightest'); }
+mark {
```

Comment:
We're not going to be able to consume this change in `okta-ui` as this will change the style of our existing components. Should be OK for now, but we might consider doing a minor version bump instead of patch.

Reply:
Just to surface this comment again - since this will break our existing style for `kbd` elements, do we mark this as a breaking change?

Reply:
@jmelberg-okta for now, I think we're only going to consider something a "breaking change" when it conflicts with the current Odyssey API use (in our case, CSS/Markup matching).

Rationale in two parts for limiting ourselves:

1. We can't control how folks are styling their raw markup _prior_ to Odyssey use or Odyssey being opinionated. If we consider this a breaking change, _any_ new styling could be considered breaking if it's not scoped to a brand new Odyssey class. Especially prior to 1.0, that seems extreme.

2. We should probably limit the scope of "breaking changes" to Odyssey UI Breakages, not UI Changes generally. While upgrading to a new version for a product might require some validation if folks have one-offs, changing styles should be expected and normalized. 

Styles that break/change functionality/behavior or represent markup changes for existing Odyssey components or official styles would definitely be considered breaking to me.

Once there's an `odyssey-web-components` package, it'll be easier to sense scope of change as well.

Reply:
Yep - this all makes sense in regards to library versioning. I wanted to just call this out as a point we need to bring up with teams who attempt to adopt this new version.